libs/core/src/components/sage-input/stories/sage-input.stories.mdx

```diff
@@ -0,0 +1,119 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+<Meta
+  title="webcomponents/Input"
+  component="sage-input"
+  argTypes={extractArgTypes('sage-input')}
+  args={{
+    disabled: false,
+    errorText: null,
+    hint: null,
+    name: null,
+    readonly: false,
+    type: 'text'
+}}
+/>
+
+export const Template = (args) => html`<sage-input
+  disabled="${args.disabled}"
+  error-text="${args.errorText}"
+  hint="${args.hint}"
+  id="${args.inputId}"
+  label="${args.label}"
+  name="${args.name}"
+  readonly="${args.readonly}"
+  type="${args.type}"
+  value="${args.value}">
+</sage-input>`;
+
+# Input
+
+<p>
+  A input field coupled with a label that allows a user to type into.
+</p>
+
+## Guidelines
+
+### Usage
+#### When to use
+
+- To gather basic usage information
+- You want a response that is one line of text
+
+#### When not to use
+- You want a response longer than one line, use a textarea (TODO: create textarea and link here)
+- You want a response from a predefined list, use a select (TODO: create select and link here)
+
+### Accessibility
+- The input component comes with a built-in label through the `label` property.   It is strongly encouraged that you include it
+- Please provide clear and descriptive error messages
+
+## Properties
+<ArgsTable of="sage-input" />
+
+## Variants
+
+### Text
+<p>The type of input used varies based on the type of information you look to capture. The `text` default type allows for any type of sinle line data.</p>
+
+<Canvas>
+  <Story name="Text" args={{ label: 'Name', type: 'text', value: 'Frank Dux' }}>
+    {Template.bind()}
+  </Story>
+</Canvas>
+
+### Email
+<p>There are several type, `text`, `email`, `file`, `number`, `password`, etc. Email is provided as a common example here.</p>
+
+<Canvas>
+  <Story name="Email" args={{ label: 'Email', type: 'email', value: 'user123@test.com' }}>
+    {Template.bind()}
+  </Story>
+</Canvas>
+
+### Disabled
+<p>Removes all interactive behavior from the text field and provides a visual treatment.</p>
+
+<Canvas>
+  <Story name="Disabled" args={{ label: 'Email', disabled: true, type: 'email', value: 'user123@test.com' }}>
+    {Template.bind()}
+  </Story>
+</Canvas>
+
+### Readonly
+<p class="my-readonly">Removes the ability to interact with the text field. If a value is present, that is still passed to the form.</p>
+
+<Canvas>
+  <Story name="Readonly" args={{ label: 'Email', readonly: true, type: 'email', value: 'user123@test.com' }}>
+    {Template.bind()}
+  </Story>
+</Canvas>
+
+### Hint
+<p>A `hint` is used to provide additional information about to complete the text field.</p>
```

Comment:
Missing the word "how":
<p>A `hint` is used to provide additional information about how to complete the text field.</p>